content/en/service_management/case_management/troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Troubleshooting
+kind: Documentation
+---
+
+## Overview
+
+This guide is intended to help you resolve issues with third-party integrations. If you continue to have trouble, reach out to [Datadog support][1] for further assistance.
```

Comment:
```suggestion
This guide is intended to help you resolve issues with third-party integrations in Case Management. If you continue to have trouble, reach out to [Datadog support][1] for further assistance.
```